CIP-0119/README.md

```diff
@@ -0,0 +1,242 @@
+---
+CIP: 119
+Title: Governance metadata - DReps
+Category: Metadata
+Status: Proposed
+Authors:
+    - Thomas Upfield <thomas.upfield@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/788
+Created: 2024-02-07
+License: CC-BY-4.0
+---
+
+## Abstract
+The Conway ledger era ushers in on-chain governance for Cardano via [CIP-1694 | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md), with the addition of many new on-chain governance artefacts. Some of these artefacts support linking to off-chain metadata as a way to provide context.
+
+The [CIP-100 | Governance Metadata](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100) standard provides a base framework for how all off-chain governance metadata should be formed and handled. But this is intentionally limited in scope, so that it can be expanded upon by more specific subsequent CIPs.
+
+This proposal aims to provide a specification for off-chain metadata vocabulary that can be used to give context to CIP-100 for DRep registration and updates. Without a sufficiently detailed standard for DRep metadata we introduce the possibility to undermine the ability of DReps to explain their motivations and therefore people looking for someone to represent them with the best possible information available to make that choice. Furthermore a lack of such standards risks preventing interoperability between tools, to the detriment of user experiences.
+
+#### Thank you
+Thank you to [everyone who participated in the CIP workshops](#Acknowledgements)
+
+## Motivation: why is this CIP necessary?
+CIP-1694 has set forth a compelling vision of a blockchain controlled by its community, and in doing so has thrown down the gauntlet to tooling providers to build apps and other kinds of tooling that will allow their users easy enjoyment of the governance features currently being built into the fabric of the Cardano blockchain. As I write this developers are feverishly hammering away at their keyboards like the blacksmiths of old to bring us the first iteration of these tools in a minimum viable format. 
```

Comment:
```suggestion
CIP-1694 has set forth a model of a blockchain controlled by its community, and in doing so has challenged tool providers to build apps and other kinds of tooling that will allow users easy access to the governance features currently being built into Cardano.  Minimum viable tools must be ready at the time these governance features are launched.
```
@Thomas-Upfield the paragraph as previously written violates a number of prohibitions we've long had in CIPs against:
- using first-person language
- writing from a personal point of view
- using subjective or dramatic language
- rooting CIPs in the time or the social conditions in which they are being written or expected to be merged / implemented.

Here is a redraft that I believe would address these problems, so please feel free to commit this version or equivalent <u>or</u> indicate what the dramatic and individualised language could accomplish behind simply identifying the requirements & time constraints of a minimum viable Governance launch.

Reply:
@rphair thanks for the rewrite. I have made the amendment.